### PR TITLE
UHF-1728 Color palette base field

### PR DESCRIPTION
**Pull request by khalima**
Instructions in this PR https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/130

Also check: 
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/130
https://github.com/City-of-Helsinki/drupal-hdbt/pull/127
https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/60

### CODE REVIEW

File: conf/cmi/core.entity_form_display.tpr_service.tpr_service.default.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ content:
   field_metatags:
     weight: 26
     settings:
-      sidebar: true
```
Comment (teroelonen) [inline conf/cmi/core.entity_form_display.tpr_service.tpr_service.default.yml:83]: Why this is changed?

**Review by teroelonen — CHANGES_REQUESTED** [review]
Looks good but I don't understand the one change that was made...

File: conf/cmi/core.entity_form_display.tpr_service.tpr_service.default.yml (hunk 1 above)
Comment (khalima) [reply conf/cmi/core.entity_form_display.tpr_service.tpr_service.default.yml:83]: This setting is applied after the the form is build (after form alter), so if it is set to true, we cannot manipulate the position of metatags field like it is done in node edit forms.

**Review by teroelonen — APPROVED** [review]
This is great.

**Pull request merged**